server/db/migrate/h2deltas/1903004_add_elastic_agent_id_to_agents.sql

```diff
@@ -0,0 +1,22 @@
+--
+-- Copyright 2019 ThoughtWorks, Inc.
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--    http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+
+-- even though the elastic profile ID definition in the config XML allows only 255 chars
+-- agents (during testing) can possibly register with more number of characters. So we keep
+-- it to 2K.
+ALTER TABLE agents ADD COLUMN elasticAgentId VARCHAR(2048) DEFAULT NULL;
+
+--//@UNDO
```

Comment:
We should add the UNDO section for all the sqls

Reply:
Also, consider adding all these DDL's in a single file? The file name should be prepended with a number which reflects the version of GoCD in which it will be released, guess `1907001`.

Reply:
Done in commit https://github.com/gocd/gocd/pull/6271/commits/d7ed13f2099cb00b83dfad2aaaa533dc4067d219